docs/user-guide.md

```diff
@@ -222,6 +252,40 @@ Configure the disk to use `/path/to/appliance.raw`
   * [Appliance README](../README.md#examples)
   * [OpenShift Documentation](https://docs.openshift.com/container-platform/4.13/installing/installing_with_agent_based_installer/preparing-to-install-with-agent-based-installer.html#installation-bare-metal-agent-installer-config-yaml_preparing-to-install-with-agent-based-installer)
   * [Static Networking](https://docs.openshift.com/container-platform/4.13/installing/installing_with_agent_based_installer/preparing-to-install-with-agent-based-installer.html#stat)
+
+#### Add custom manifests (Optional)
+* Note that any manifest added here will apply to **only** to the cluster installed using this config-iso.
+* Find more details and additional examples in OpenShift documentation: [Customizing nodes](https://docs.openshift.com/container-platform/4.13/installing/install_config/installing-customizing.html).
+
+1. Create the openshift manifests directory
+```shell
+mkdir $APPLIANCE_ASSETS:/openshift
+```
+
+2. Add one or more custom manifests under `$CLUSTER_CONFIG:/openshift`. For example:
+```yaml
```

Comment:
Not sure we need another example here. The docs should cover it for the user flow.
Just add also this reference for 'Post-installation configuration': https://docs.openshift.com/container-platform/4.13/post_installation_configuration/machine-configuration-tasks.html#using-machineconfigs-to-change-machines

Reply:
Done

Reply:
Added a cross reference to the above example instead.